server/controllers/finance/reports/balance/index.js

```diff
@@ -223,5 +223,32 @@ function balanceReporting(params) {
   .then(function (rows) {
     data.middle = rows;
     return data;
+  })
+  .then(function (rows) {
+    // Manual mixing 
+    const TITLE_ACCOUNT_TYPE = 4;
```

Comment:
This is a good declaration :+1:

In the future, when you have a constant like this, it would be better to put it outside the function, in the main body of the module.  That way, it is not re-defined every single HTTP request.  Like this:

```js
const TITLE_ACCOUNT_TYPE = 4;

function balanceReporting() {
  // something something batman...
}
```